README.md

```diff
@@ -3,128 +3,95 @@
+Open a terminal and execute the following commands:
 
 ```
-$ cd ~/path/to/cloned/ansible-playbook-airflow
-$ pipenv shell
-(ansible-playbook-airflow) $ make up
-# update `data/example-variables.json` to add whatever Airflow variables you want; you can also do this in the GUI if preferred.
-# update `Makefile` to add whatever Airflow variables you want; you can also do this in the GUI if preferred.
-# check http://localhost:8010 is running okay
-(ansible-playbook-airflow) $ make update
+$ git clone https://github.com/tulibraries/funcake_dags.git
+$ cd funcake_dags
+$ make up
 ```
 
-Then change into `dags/funcake_dags` and `git pull origin your-working-branch` to get your working branch locally available. *Symlinks will not work due to how Airflow & Docker handle mounts.* You could manually copy over files if that works better for you.
+Wait for Airflow and associated servers to build and start up. This may take several minutes.
 
-Continuing from above:
+Open a browser and visit http://localhost:8010 to verify  Airflow server is running.
 
-```
-(ansible-playbook-airflow) $ cd dags
-(ansible-playbook-airflow) $ git clone https://github.com/tulibraries/funcake_dags.git
-(ansible-playbook-airflow) $ cd funcake_dags
-(ansible-playbook-airflow) $ git checkout -b my-working-branch
-```
+On initial startup, the dashboard may display an empty or partial list of DAGs and the status at the top may show the Broken DAG error message indicating that a connection or variable. Create the connection and copy it's attributes from the [TUL Production Airflow Server Connections](http://localhost:8010/admin/connection/) or [TUL QA Airflow Server Connections](http://localhost:8010/admin/connection/).  Create the variable and copy it's value from the [TUL Production Airflow Server Variables](http://localhost:8010/admin/variable/) or [TUL QA Airflow Server Variables](http://localhost:8010/admin/variable/).
 
-Changes to this DAGs folder will be reflected in the local Airflow web UI and related Airflow services within a few seconds. *DAG folders in the ansible-playbook-airflow directory will not be replaced, updated, or touched by any make commands if they already exist.*
+Specific Make targets and their descriptions are documented in [Airflow Docket Dev Setup](airflow-docker-dev-setup/README.md).
 
-When you're done with local development, git commit & push your changes to funcake_dags up to your working branch, and tear down the docker resources:
 
-```
-(ansible-playbook-airflow) $ make down
-(ansible-playbook-airflow) $ exit # leaves the pipenv virtual environment
-```
+### Related Documentation and Projects
+
+- [Airflow Docker Development Setup](https://github.com/tulibraries/airflow-docker-dev-setup)
+- [Ansible Playbook Airflow](https://github.com/tulibraries/ansible-playbook-airflow)
+- [Apache Airflow](https://airflow.apache.org/docs/)
+- [CircleCI](https://circleci.com/docs/2.0/configuration-reference/)
+
 
 ## Linting & Testing
```

Comment:
We have commands in the airflow-docker-dev-setup Makefile for both testing and listing.  You might want to mention that in this section.

Reply:
Added.  But Makefile lints on files in a .dag subdirectory.  Tests, OTOH are available with the airflow-docker-dev-setup Makefile